docs/support_files/api_reference.md

```diff
@@ -89,12 +87,13 @@ Aliases: `Given`, `When`, `Then`.
   - Should have one argument for each capture in the regular expression.
   - May have an additional argument if the gherkin step has a docstring or data table.
   - When using the asynchronous callback interface, have one final argument for the callback function.
+* `phase`: A string equal to `'given'`, `'when'`, `'then'`, or `undefined`.
```

Comment:
Can we move this to `options.phase`? I'd prefer not to add a parameter after a callback

Reply:
👍done